### DISCUSSION TITLE
Cannot build deep-kit_docker

### DISCUSSION THREAD

**Discussion by LuisStudio** [deepmodeling/deepmd-kit#3] [Q&A] [answered]
Hi!
I want to build the deepmd-kit_docker but met the following error :

Starting local Bazel server and connecting to it...
.............
ERROR: Unrecognized option: --incompatible_load_argument_is_label=false
The command '/bin/sh -c cd /root/tensorflow && ./configure < install_input && bazel build -c opt --incompatible_load_argument_is_label=false --copt=-msse4.2 --verbose_failures //tensorflow:libtensorflow_cc.so' returned a non-zero code: 2

I tried to delete the '--incompatible_load_argument_is_label=false', I met the following error:
Starting local Bazel server and connecting to it...
...............
Loading:
Loading: 0 packages loaded
Loading: 0 packages loaded
ERROR: /root/tensorflow/WORKSPACE:15:1: Traceback (most recent call last):
File "/root/tensorflow/WORKSPACE", line 15
closure_repositories()
File "/root/.cache/bazel/_bazel_root/efb88f6336d9c4a18216fb94287b8d97/external/io_bazel_rules_closure/closure/repositories.bzl", line 69, in closure_repositories
_check_bazel_version("Closure Rules", "0.4.5")
File "/root/.cache/bazel/_bazel_root/efb88f6336d9c4a18216fb94287b8d97/external/io_bazel_rules_closure/closure/repositories.bzl", line 172, in _check_bazel_version
fail(("%s requires Bazel >=%s but was...)))
Closure Rules requires Bazel >=0.4.5 but was 0.14.1- (@non-git)
ERROR: Error evaluating WORKSPACE file
ERROR: Skipping '//tensorflow:libtensorflow_cc.so': error loading package 'external': Package 'external' contains errors
WARNING: Target pattern parsing failed.
ERROR: error loading package 'external': Package 'external' contains errors
INFO: Elapsed time: 2.177s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)
The command '/bin/sh -c cd /root/tensorflow && ./configure < install_input && bazel build --config=opt --copt=-msse4.2 --verbose_failures //tensorflow:libtensorflow_cc.so' returned a non-zero code: 1

So I guess the reason is about the updated version of bazel these days but I don't know how to install the old version of bazel in docker. Could you update the docker or tell me how to fix this problem?
Thank you so much.

**Comment by frankhan91** [comment, marked as answer]
You can try to use this docker first: https://github.com/frankhan91/deepmd-kit_docker.
It worked one month ago.
